### DISCUSSION TITLE
Changing player voice volume via script

### DISCUSSION THREAD

**Discussion by taro8** [IDI-Systems/acre2#1238] [Q&A] [answered]
Hello.
I'm looking for a way to change player voice volume using a script. I want to run a script on a player and have his voice volume set to max.
I did some digging through the code, but I was unable to find something concrete. The best I could found is `acre_sys_gui_volumelevel` global variable, but I have suspicion it would change the volume for all players if I change it in MP.

**Comment by Drofseh** [comment, marked as answer]
`acre_sys_gui_volumeLevel` is for the colour of the volume bar, `acre_api_fnc_setSelectableVoiceCurve` will set the actual volume.
Both are local.
Relative values:
```
acre_sys_gui_volumeLevel | acre_api_fnc_setSelectableVoiceCurve
                       0 | 0.1
                       1 | 0.4
                       2 | 0.7
                       3 | 1
                       4 | 1.3
```
For shouting:
```sqf
[1.3] call acre_api_fnc_setSelectableVoiceCurve;

[{
    acre_sys_gui_volumeLevel = 4;
}, [], 1] call CBA_fnc_waitAndExecute;
```
Not sure if the delay to change the bar colour is required, but it's what I wrote ages ago.
